integration-tests/src/main/java/io/druid/testing/clients/CoordinatorResourceTestClient.java

```diff
@@ -70,7 +70,7 @@ private String getCoordinatorURL()
   {
     Map<String, Integer> status = null;
     try {
-      StatusResponseHolder response = makeRequest(HttpMethod.GET, getCoordinatorURL() + "loadstatus?simple");
+      StatusResponseHolder response = makeRequest(HttpMethod.GET, getCoordinatorURL() + "loadstatus");
```

Comment:
Since this is being touched, can you make it use the URL building methods instead of string building?